client/landing/jetpack-cloud/sections/backups/rewind-flow/download.tsx

```diff
@@ -0,0 +1,190 @@
+		[ dispatch, rewindConfig, rewindId, siteId ]
+	);
+
+	const renderConfirm = () => (
+		<>
+			<div className="rewind-flow__header">
+				<img
+					src="/calypso/images/illustrations/jetpack-cloud-download-ready.svg"
+					alt="jetpack cloud download ready"
+				/>
+			</div>
+			<h3 className="rewind-flow__title">{ translate( 'Create downloadable backup' ) }</h3>
+			<p className="rewind-flow__info">
+				{ translate(
+					'{{strong}}%(downloadTimestamp)s{{/strong}} is the selected point to create a download backup of. ',
+					{
+						args: {
+							downloadTimestamp,
+						},
+						components: {
+							strong: <strong />,
+						},
+					}
+				) }
+			</p>
+			<h4 className="rewind-flow__cta">{ translate( 'Choose the items you wish to restore:' ) }</h4>
+			<RewindConfigEditor currentConfig={ rewindConfig } onConfigChange={ setRewindConfig } />
+			<RewindFlowNotice
+				gridicon="notice-outline"
+				title={ translate( 'More info' ) }
+				type={ RewindFlowNoticeLevel.NOTICE }
+				link="https://jetpack.com/support/backup"
+			/>
+			<Button
+				className="rewind-flow__primary-button"
+				primary
+				onClick={ requestDownload }
+				disabled={ Object.values( rewindConfig ).every( setting => ! setting ) }
+			>
+				{ translate( 'Create downloadable file' ) }
+			</Button>
+		</>
+	);
+
+	const renderInProgress = () => (
+		<>
+			<div className="rewind-flow__header">
+				<img
+					src="/calypso/images/illustrations/jetpack-cloud-download-ready.svg"
+					alt="jetpack cloud download ready"
+				/>
+			</div>
+			<h3 className="rewind-flow__title">
+				{ translate( 'Currently creating a downloadable backup of your site' ) }
+			</h3>
+			<ProgressBar percent={ downloadProgress } />
+			<p className="rewind-flow__info">
+				{ translate(
+					"We're creating a downloadable backup of your site from {{strong}}%(downloadTimestamp)s{{/strong}}.",
+					{
+						args: {
+							downloadTimestamp,
+						},
+						components: {
+							strong: <strong />,
+						},
+					}
+				) }
+			</p>
+			<CheckYourEmail
+				message={ translate( "For your convenience, we'll email you when your file is ready." ) }
+			/>
+		</>
+	);
+
+	const renderReady = () => (
+		<>
+			<div className="rewind-flow__header">
+				<img
+					src="/calypso/images/illustrations/jetpack-cloud-download-success.svg"
+					alt="jetpack cloud download success"
+				/>
+			</div>
+			<h3 className="rewind-flow__title">
+				{ translate( 'Your backup is now available for download.' ) }
+			</h3>
+			<p className="rewind-flow__info">
+				{ translate(
+					'We successfully created a backup of your site from {{strong}}%(downloadTimestamp)s{{/strong}}.',
+					{
+						args: {
+							downloadTimestamp,
+						},
+						components: {
+							strong: <strong />,
+						},
+					}
+				) }
+			</p>
+			<Button href={ downloadUrl } primary className="rewind-flow__primary-button">
+				{ translate( 'Download file' ) }
+			</Button>
+			<CheckYourEmail
```

Comment:
This doesn't seem relevant. 
Instead of "Check your email" heading we should probably something different. 

Like "Email Sent" for example.
...

Reply:
tagging design for review